graphql_compiler/tests/snapshot_tests/test_cost_estimation.py

```diff
@@ -1786,7 +1786,7 @@ def test_int_value_conversion_uuid(self):
             "80000000-0000-",
         ]
         for uuid_value in invalid_uuid_values:
-            with self.assertRaises(Exception):
+            with self.assertRaises(Union[AssertionError, NotImplementedError]):
```

Comment:
I think this should be a tuple rather than a `Union` type: https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertRaises